tasks/task.json

```diff
@@ -100,6 +100,10 @@
             "description": "Whether the task runner should treat the parameter value as sensitive",
             "type": "boolean",
             "default": false
+          },
+          "position": {
+            "description": "The position the parameter should be passed to the script",
```

Comment:
Does this start with 0 or 1 indexing? Is it an error if sequential parameters are not declared in metadata, or provided when running the command? Are all positional parameters required?

Reply:
I'd put those details in the Task Parameters > Options section when describing the `position` property.

Reply:
To be flexible the scheme could output parameters sorted on position/order. That means it could use 10, 20, 30 etc to allow for parameters to be inserted in the future - i.e. less
busy work if another positional parameter needs to be injected somewhere in the middle.
The specification also needs to define what happens if some parameters have position/order and others don't. For example; positional parameters are given first (in order) and the remaining parameters are output in undefined order.